src/coredata.cs

```diff
@@ -2346,6 +2364,14 @@ partial interface NSPersistentStoreCoordinator
 		[Export ("currentPersistentHistoryTokenFromStores:")]
 		[return: NullAllowed]
 		NSPersistentHistoryToken GetCurrentPersistentHistoryToken ([NullAllowed] NSObject [] stores);
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("finishDeferredLightweightMigration:")]
+		bool FinishDeferredLightweightMigration ([NullAllowed] out NSError error);
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("finishDeferredLightweightMigrationTask:")]
+		bool FinishDeferredLightweightMigrationTask ([NullAllowed] out NSError error);
```

Comment:
These are old versions too but looks like they may actually be older? Just checking

Reply:
This ones are correct. We probably forgot them in one of the . release.